WordPress/Classes/Extensions/WKWebView+UserAgent.swift

```diff
@@ -29,4 +30,16 @@ extension WKWebView {
     static func userAgent() -> String {
         return WKWebView().userAgent()
     }
+
+    /// It makes the first render of the next `WKWebView` x2-3 times faster.
+    static func warmup() {
+        let renderer = WebCommentContentRenderer()
```

Comment:
Should this function be `WebCommentContentRenderer.warmup()`?

Reply:
It put it as an extension for `WKWebView` because it effectively warms up `WebKit` itself and it's not comment-specific. For example, it also affects the editor and the many other web flows. `WebCommentContentRenderer` is an implementation detail.